app/design/frontend/Gumennoy/FirstTheme/Magento_Newsletter/templates/js/components.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
```

Comment:
Seems this file is the same as the original one: https://github.com/magento/magento2/blob/2.2/app/code/Magento/Newsletter/view/frontend/templates/js/components.phtml
Is it plannet to be changed in future? If not - please remove this file from your theme

Reply:
File removed, It was created for test